src/pages/index.js

```diff
@@ -81,45 +158,106 @@ const newCardPopup = new PopupWithForm({
 });
 newCardPopup.setEventListeners();
 
+const userInfo = new UserInfo({
+  profileTitleSelector: ".profile__title",
+  profileDescriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+Promise.all([api.getUserInfo(), api.getInitialCards()]);
```

Comment:
Needs correction: Please remove this line, it does nothing. If you want to use Promise.all then you should do that for receiving the data instead of lines 167-176 and 134-142:

```js
Promise.all([api.getUserInfo(), api.getInitialCards()])
  .then(([userInfoResponse, initialCardsResponse]) => {
    // update the user info using userInfoResponse
    // set the initial cards using initialCardsResponse
  })
  .catch(..)
```